e2e/tools/validator/scripts/stressor.sh

```diff
@@ -2,29 +2,16 @@
 
 # warm up system by waiting for 40 seconds
 # idle
-stress-ng --cpu $(nproc) --cpu-load 0 --timeout 10s
+cpu=$(nproc)
 
-# # Stress at 10% load for 20 seconds
-# stress-ng --cpu $(nproc) --cpu-load 10 --timeout 20s
-#
-# # Stress at 25% load for 20 seconds
-# stress-ng --cpu $(nproc) --cpu-load 25 --timeout 20s
-#
-# # Stress at 50% load for 20 seconds
-# stress-ng --cpu $(nproc) --cpu-load 50 --timeout 20s
-#
-# Stress at 75% load for 10 seconds
-stress-ng --cpu $(nproc) --cpu-load 75 --timeout 20s
-#
-# # Stress at 50% load for 20 seconds
-# stress-ng --cpu $(nproc) --cpu-load 50 --timeout 20s
-#
-# # Stress at 25% load for 20 seconds
-# stress-ng --cpu $(nproc) --cpu-load 25 --timeout 20s
-#
-# # Stress at 10% load for 20 seconds
-# stress-ng --cpu $(nproc) --cpu-load 10 --timeout 20s
-
-# cool off for 40 seconds
-# idle
-stress-ng --cpu $(nproc) --cpu-load 0 --timeout 10s
+stress-ng --cpu "$cpu" --cpu-load 0 --timeout 10s # idle for 2 scrapes
```

Comment:
how about this one?
```bash
 for level in $(seq 10 10 100); do
              stress-ng --cpu $(nproc) --cpu-load $level --timeout 10s
```

Reply:
https://gist.github.com/sthaha/fc5c33ae6f92945902296fa665b41447 is what I run on my VM

The idea is to have 100% running for longer

Reply:
fixed now